README.md

```diff
@@ -372,3 +372,12 @@ ahead to the failed step and try to rerun it.
     (https://cloud.google.com/genomics/v1alpha2/pipelines).
 *   Read about [Dataflow](https://cloud.google.com/dataflow).
 *   Write your own workflows!
+
+## A word about quotas
+
+Google has numerous quotas that may impact the parallel execution of large numbers of tasks.  Some hints include:
+
+* Check and potentially increase quotas here: https://console.cloud.google.com/compute/quotas
```

Comment:
[Check and potentially increase quotas](https://console.cloud.google.com/compute/quotas)